### DISCUSSION TITLE
Virtualenv using the wrong folder dir to find python version

### DISCUSSION THREAD

**Discussion by abhijit-inge** [google/or-tools#3529] [Build and configuration issues]
Hello,

i'm building ortools for linux and I added virtualenv as a dependency. However while building the package i see the following error 
`<wrking dir>/build/python/venv/bin/real/python3.9: cannot open shared object file: No such file or directory`

I believe this is because real doesn't exist in the path to python3.9. However ven/bin/python3.9 exists. Any way to mitigate this issue? We are using 
`virtualenv v20.x`
`Ortools branch 9.4`
`python3.x`
we ran the following commands
```
cmake -S. -Bbuild -DBUILD_DEPS:BOOL=ON -DBUILD_PYTHON:BOOL=ON
cmake --build build --target python_package -v
```

**Comment by Mizux** [comment]
1) Which distro did you use ? (I'm curious of why you have a `/real/` in your venv path)
2) Can you provide a longer/full build trace to better know at which step exactly it fails and have more context...
